blueprints/cloud-operations/network-dashboard/variables.tf

```diff
@@ -50,28 +50,35 @@ variable "organization_id" {
 
 variable "prefix" {
   description = "Customer name to use as prefix for monitoring project"
+  default     = "mon"
```

Comment:
We usually don't provide defaults for prefix

Reply:
kk